sdkproject/Assets/Mapbox/Unity/Constants.cs

```diff
@@ -39,6 +39,7 @@ public static class Math
 			public static readonly Vector3 Vector3Down = new Vector3(0, -1, 0);
 			public static readonly Vector3 Vector3One = new Vector3(1, 1, 1);
 			public static readonly Vector3 Vector3Forward = new Vector3(0, 0, 1);
+			public static readonly Vector3 Vector3Null = new Vector3(9999.99f, 9999.99f, 9999.99f);
```

Comment:
Lets call this Vector3NotInitialized/Vector3Unused/Vector3LargeValue or something explicit. Null is misleading. 
Also, lets use floatmax instead of 9999.99

Reply:
Changing it to Vector3Unused and going with float.minvalue instead of max.